systemtests/src/test/java/io/enmasse/systemtest/iot/SimpleK8sDeployTest.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.enmasse.systemtest.iot;
+
+import static io.enmasse.systemtest.TestTag.sharedIot;
+import static io.enmasse.systemtest.TestTag.smoke;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.enmasse.systemtest.Environment;
+import io.enmasse.systemtest.Kubernetes;
+import io.enmasse.systemtest.TestUtils;
+import io.enmasse.systemtest.TimeoutBudget;
+
+@Tag(sharedIot)
+@Tag(smoke)
+public class SimpleK8sDeployTest {
+
+    private static final String NAMESPACE = Environment.getInstance().namespace();
+
+    private static final String[] EXPECTED_DEPLOYMENTS = new String[] {
+                    "iot-auth-service",
+                    "iot-device-registry",
+                    "iot-gc",
+                    "iot-http-adapter",
+                    "iot-mqtt-adapter",
+                    "iot-tenant-service",
+    };
+
+    private static final Map<String,String> IOT_LABELS;
+
+    static {
+        IOT_LABELS= new HashMap<> ();
+        IOT_LABELS.put("component", "iot");
+    }
+
+    private Kubernetes client = Kubernetes.getInstance();
+
+    private static void execute(final String [] command ) throws InterruptedException, IOException {
+        Process p =  new ProcessBuilder(command)
+                .inheritIO()
+                .start();
+
+        final int rc = p.waitFor();
+        if ( rc != 0) {
+            throw new RuntimeException(String.format("Failed to run: %s -> %s", command, rc));
+        }
+    }
+
+    @BeforeAll
+    protected static void setup () throws Exception {
+        execute(new String[] {
```

Comment:
@ctron why do you use execution like this? We have executor in systemtest which can work with oc and kubectl

Reply:
I didn't know and find … I will migrate that then.

Reply:
Or better is it posible to domit via fabric8 kubernetes client.

Reply:
True, can you apply a JSON file with it?

Reply:
So applying a JSON file doesn't seem to work, as it must have a Java implementation of the JSON classes … 

I would like to simply load that using `kubectl` … however the only `execute` method I found was tied to `minikube`.

Reply:
Ok, found something. Replaced it with `KubeCMDClient`.